src/app/container/container.component.html

```diff
@@ -27,13 +27,16 @@
   </div>
 </div>
 <div class="row">
-  <div class="col-md-7">
+  <div class="col-md-12">
```

Comment:
This pushes the publish and refresh buttons down in another row below the title.   Not sure if that's intended

Reply:
Oops